working-examples/device-motion-sensor-input/index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html>
+<head>
+<style type="text/css">
+
+.slidecontainer {
+  width:100%;
+}
+</style>
+
+<title>WCAG Example of Motion Sensor</title> 
+</head>
+<body>
+
+<pre class="output"></pre>
+
+  <h1>Slider Motion Sensor Example </h1>
+
+ <p>Open this slider in a mobile device such as a smart phone or tablet. Tilt the phone to the right and left to adjust the slider value. The decrease and increase buttons also adjust the value.  The check box disables the motion sensing adjustment.</p>

```

Comment:
you also have non-mobile devices, such as laptops, with motion sensors. maybe "Open this slider on a device with a motion sensor, such as a smart phone or tablet. Tilt the device..."